components/GroupMultipleQ.js

```diff
@@ -22,26 +22,26 @@ export default function GroupMultipleQ({ question }) {
                 title={
                   <Text>{question.question.items[0].choices[0].label}</Text>
                 }
-                // checked={}
-                // onPress={}
+              // checked={}
```

Comment:
Since we found out we could reuse the GroupSingle component for the GroupMultiple questions also, we can't forget to get rid of this file at some point. Great thinking!